### DISCUSSION TITLE
Restrict the "add a new cable" options to same site

### DISCUSSION THREAD

**Discussion by llamafilm** [netbox-community/netbox#15210] [Ideas]
When I go to add an ethernet cable between 2 interfaces, I see a massive list of _all_ devices to choose from.  The selector dialog helps, but it takes time to click through all the fields.  I propose that this list should be filtered by default to show only devices in the same site.  I never connect interfaces between sites, so this would work for me 100% of the time.  I imagine other people probably do connect between sites but I'm guessing that would be relatively rare.  So in that case, you would just have to open the selector dialog and deselect the filter.  I'd love to hear others' opinions on this idea.

**Comment by llamafilm** [comment]
I just realized this was requested in #2897 and closed with no resolution.

**Comment by Chiniquy** [comment]
Yes please. Cabling from devices to generic unnamed ODF / patchpanels is a pain atm, since you cant search for them by name.
